Earthfile

```diff
@@ -435,12 +435,12 @@ node-ci-image-single-platform:
         protobuf-compiler \
         pkg-config \
         grcov \
-        openssh-client \
-        gcc-aarch64-linux-gnu \
-        libc6-dev-arm64-cross \
-        gcc-x86-64-linux-gnu \
-        crossbuild-essential-amd64 \
-        libc6-amd64-cross
+        openssh-client
+        # gcc-aarch64-linux-gnu \
+        # libc6-dev-arm64-cross \
+        # gcc-x86-64-linux-gnu \
+        # crossbuild-essential-amd64 \
+        # libc6-amd64-cross
```

Comment:
Want to keep theses comments?

Reply:
I want to keep them in case anyone wants to cross compile (for whatever reason).